pkg/requestmeta/requestmeta.go

```diff
@@ -29,6 +29,13 @@ const (
 	// the New Enemy Problem. This is only used with the CockroachDB datastore,
 	// and only if user-provided request overlap is enabled.
 	RequestOverlapKey RequestMetadataHeaderKey = "io.spicedb.requestoverlapkey"
+
+	// RequestIDKey, if specified in a request header, will propagate the given string value
+	// through SpiceDB for the lifetime of the request. This can be used to correlate logs
+	// and traces with a specific request.
+	//
+	// TODO(alecmerdler): This is duplicated in SpiceDB source; we should be importing it from here.
```

Comment:
nit: this seems like the wrong place for this TODO since it needs to change in a different project